### PR TITLE
Add MVP home & about pages behind switch

### PR DESCRIPTION
**Pull request by stephaniehobson**
## One-line summary

Add MVP home and about pages.

## Significant changes and points to review

- Add home page for English only behind switch `m24-home`
- Add about page for English only behind switch`m24-about`
- Add beta font files for review purposes (we can remove before merging)
- Move grid variables to a new file

## Issue / Bugzilla link

#14893 
#14892 

## Testing

- Copy and images are not final or are missing, please focus this review on the code.
- Please [test the switches](https://bedrock.readthedocs.io/en/latest/install.html#testing) locally.

### CODE REVIEW

**Comment by stephaniehobson** [comment]
Doing my own review:

- [ ] consider component cta link has a too wide outline
- [ ] consider component cta not right aligned on mobile
- [ ]  consider component cta needs more space above it on mobile
- [ ]  consider component image is awkward at 820 (center? max-width?)
- [ ]  consider component 50/50 split is awkward at 1024 with no text on right
- [ ] font not showing up in Safari
- [ ] add links where known
- [ ] when touching/tapping in the product/spring list there's some weirdness in hover/focus states
- [ ] lines appearing between rows in transition component (maybe only a bug in Chrome responsive design view?)
- [ ] crawlbars at multiple breakpoints Chrome (390, 480, 768, 1024)

Recommendation: stop trying to use the same component for both the donate and careers blocks

**Comment by stephaniehobson** [comment]
Switched the body font to semi-bold.

File: bedrock/mozorg/templates/mozorg/home/includes/m24/donate.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <div class="m24-c-donate-media"><img src="{{ static('img/m24/todo-27-1.svg') }}" alt="" height="253" width="680"></div>
+    <div class="m24-c-donate-info">
+      <p class="m24-c-donate-info-title">Where do your dollars go?</p>
+      <ul class="mzp-u-list-styled">
+        <li>Funding new AI privacy technologies</li>
+        <li>Building innovation labs</li>
+        <li>Sponsoring incoming 2025 Fellows</li>
+      </ul>
+    </div>
+    <p class="m24-c-donate-cta"><a href="#TODO" class="m24-c-cta" data-cta-text="Donate">Donate</a></p>
```
Comment (reemhamz) [inline bedrock/mozorg/templates/mozorg/home/includes/m24/donate.html:19]: **Nit:** This CTA should be left aligned on mobile:

<kbd>Screenshot 2024-09-10 at 6 26 05 PM</kbd>

File: bedrock/mozorg/templates/mozorg/home/includes/m24/products.html
Comment (reemhamz) [inline bedrock/mozorg/templates/mozorg/home/includes/m24/products.html:1]: **Praise:** I love how this section looks! You did a great job developing it.

File: media/css/m24/accordion.scss
Comment (reemhamz) [inline media/css/m24/accordion.scss:1]: I don't see any HTML content with the classnames mentioned in this file?

**Review by reemhamz — APPROVED** [review]
oof, this was a big review!
I've left some drive-by comments to be addressed + some questions.
Overall, this is looking pretty good for what we have so far. We should address the bad color contrast on hover links before launching MVP but I know we're waiting on a few more updates on that end.
Great job! r+wc

File: bedrock/mozorg/templates/mozorg/home/includes/m24/showcase.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+<div class="m24-t-green">
+  <section class="m24-c-showcase">
+    <h2 class="m24-c-showcase-title">Lorem ipsum dolor sit amet</h2>
+    <p class="m24-c-showcase-subtitle">Suspendisse commodo sed erat in pellentesque. In sit amet tortor porttitor, gravida mauris tempor, pharetra erat. Nam vel justo urna.</p>
+    <div class="m24-c-showcase-media">
+        <img src="{{ static('img/m24/todo-1-1.svg') }}" alt="" height="500" width="500">
+    </div>
+    <p class="m24-c-showcase-info">Sed eu mattis</p>
+    <p class="m24-c-showcase-cta"><a href="#TODO" class="m24-c-cta" data-cta-text="">Learn more</a></p>
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/mozorg/templates/mozorg/home/includes/m24/showcase.html]: No colours are final but this is still a good call out. Specially where the CTA is used on purple the CTA links need to not use the default hover states.

File: media/css/m24/consider.scss [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+.m24-c-consider-media {
+    img {
+        display: inline-block;
+        vertical-align: bottom;
+        margin-left: $grid-gutter;
+
+        &:first-child {
+            margin-left: 0;
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply media/css/m24/consider.scss]: ✅

File: media/css/m24/copy.scss [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    }
+}
+
+.m24-c-copy-title {
+    font-size: $alias-text-title-h2;
+    margin-bottom: 0;
+    padding-bottom: $spacer-sm;
+    text-wrap-style: balance;
+
+    &.m24-t-2xl{
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply media/css/m24/copy.scss:24]: ✅

File: bedrock/mozorg/templates/mozorg/about/includes/m24/careers.html [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+
+<div class="m24-c-consider">
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/mozorg/templates/mozorg/about/includes/m24/careers.html:15]: I'm hoping we can use these components more generally in the future. 🤞

File: bedrock/mozorg/templates/mozorg/home/home-m24.html [hunk 6]
```diff
@@ -0,0 +1,51 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros-protocol.html" import split, billboard, picto with context %}
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/mozorg/templates/mozorg/home/home-m24.html:7]: ✅

File: bedrock/mozorg/templates/mozorg/home/includes/m24/products.html
Comment (stephaniehobson) [reply bedrock/mozorg/templates/mozorg/home/includes/m24/products.html:1]: ❤️

File: media/css/m24/accordion.scss
Comment (stephaniehobson) [reply media/css/m24/accordion.scss:1]: Oops, yeah, I was messing around with other components but we don't need this one for anything yet.

File: bedrock/mozorg/templates/mozorg/about/includes/m24/careers.html [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+  <h2 class="m24-c-consider-title">Mozilla jobs are where skill meets passion</h2>
+  <div class="m24-c-consider-media">
+    <img src="{{ static('img/m24/todo-2-3.svg') }}" alt="" height="324" width="216">
+    <img src="{{ static('img/m24/todo-2-3.svg') }}" alt="" height="324" width="216">
+    <img src="{{ static('img/m24/todo-2-3.svg') }}" alt="" height="324" width="216">
+    <img src="{{ static('img/m24/todo-2-3.svg') }}" alt="" height="324" width="216">
+  </div>
+  <p class="m24-consider-cta-info">Interested in advocacy? Love coding? Join our best-in-class talent and let’s transform the web, for everyone.</p>
+  <div class="m24-c-consider-cta">
+    <a href="#TODO" class="m24-c-cta" data-cta-text="Careers">Check out jobs</a>
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/mozorg/templates/mozorg/about/includes/m24/careers.html]: ✅

File: bedrock/mozorg/templates/mozorg/home/includes/m24/showcase.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/mozorg/templates/mozorg/home/includes/m24/showcase.html]: Colours are updated :)

File: bedrock/mozorg/templates/mozorg/about/includes/m24/leadership.html [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+#}
+
+<div class="m24-t-green">
+  <section class="m24-c-showcase">
+    <h2 class="m24-c-showcase-title">Leadership ipsum dolor sit amet</h2>
+    <p class="m24-c-showcase-subtitle">Suspendisse commodo sed erat in pellentesque. In sit amet tortor porttitor, gravida mauris tempor, pharetra erat. Nam vel justo urna.</p>
+    <div class="m24-c-showcase-media">
+      <img src="{{ static('img/m24/todo-1-1.svg') }}" alt="" height="500" width="500">
+    </div>
+    <p class="m24-c-showcase-cta"><a href="#TODO" class="m24-c-cta" data-cta-text="Leadership">Leadership</a></p>
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/mozorg/templates/mozorg/about/includes/m24/leadership.html]: Colours are updated :)

**Pull request merged**